src/pages/hardware/developingAnIOX/MIMEProtocol.tsx

```diff
@@ -0,0 +1,294 @@
+        },
+    "isDirectionToVehicle": true,
+    "messageSize": 235 // If unspecified defaults to 235. Max of 1000.
+    },
+}, function(result) {
+    console.log("Done: ", result);
+}, function(e) {
+    console.error("Failed:", e);
+});
+`}/>
+        <h4>Retrieving MIME Data From MyGeotab:</h4>
+        <p>Once the data has been sent to the cloud, you can use the MyGeotab SDK to pull the message content out of the target database. For each MIME payload, the data is stored within a single TextMessage Object and can be retrieved using the Get method (search criteria can be added):</p>
+        <CodeSample
+        language="javascript"
+        code={`api.call("Get", {
+    "typeName":"TextMessage",
+    "resultsLimit":10
+});
+`}/>
+        <p>The payload is stored in MyGeotab in Base64 encoded format. Decoding this payload from Base64 to bytes should yield the expected content which was passed through. In the event that the database has multiple devices sending MIME data at different times, you may want to specify the originating device and/or filter by the time interval during which MIME data was sent:</p>
+        <CodeSample
+        language="javascript"
+        code={`api.call("Get", { // First API call is to get the ID of a device given its serial number
+    "typeName": "Device",
+    "search": {
+        "serialNumber": "G9##########" // Replace with your GO device serial number
+    },
+    "resultsLimit": 1
+}, function(deviceResult) {
+    console.log("Found device ID: ", deviceResult[0].id);
+    api.call("Get", { // Second API call is to get all TextMessages sent to/from this device
+        "typeName": "TextMessage",
+        "search": {
+            "deviceSearch": {"id": deviceResult[0].id }, // Optional: Return only TextMessages sent to/from this device
+            "modifiedSinceDate": "2023-09-01T14:00:00.000Z", // Optional: Specify the minimum datetime for messages
+            "toDate": "2023-09-20T18:30:00.000Z", // Optional: Specify the maximum datetime for messages
+            "contentTypes": ["MimeContent"], // Optional: Return only MimeContent
+            "isDirectionToVehicle": false // Optional: Return only messages sent from MyGeotab to IOX (true) or from IOX to MyGeotab (false)
+        },
+        "resultsLimit": 100
+    }, function(textMessageResult) {
+        console.log("TextMessages: ", textMessageResult);
+        if (textMessageResult.length > 0) {
+            console.log("base64 data for first TextMessage: ", textMessageResult[0].messageContent.data);
+        }
+    }, function(textMessageError) {
+        console.error("TextMessage search failed: ", textMessageError);
+    });
+}, function(deviceError) {
+    console.error("Device search failed:", deviceError);
+});
+`}/>
+    </div>
+);
+
+const pageTitle: PageTitleProps = {
+    "title": "MIME Protocol",
+    "breadCrumbItems": ["Hardware", "Developing an IOX", "MIME Protocol"]
```

Comment:
```suggestion
    "breadCrumbItems": ["Hardware", "Guides", "MIME Protocol"]
```
There won't be a "Developing an IOX" section in the new site, so let's replace it with "Guides". Can you also update the folder name to 'guides'?